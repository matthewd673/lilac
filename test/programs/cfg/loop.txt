L1:
jz 0 L1
