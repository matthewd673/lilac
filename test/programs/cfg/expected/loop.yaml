---
nodes:
- ENTRY
- BB_1
- EXIT
edges:
- from: ENTRY
  to: BB_1
- from: BB_1
  to: BB_1
- from: BB_1
  to: EXIT
